src/main/java/org/logstash/dissect/fields/AppendField.java

```diff
@@ -0,0 +1,52 @@
+package org.logstash.dissect.fields;
+
+import com.logstash.Event;
+import org.logstash.dissect.ValueResolver;
+
+import java.util.Map;
+
+public final class AppendField extends AbstractField {
+    private static final int ORD = 2;
+
+    private AppendField(String s, int ord) {
+        super(s, ord);
+    }
+
+    public static Field create(String s) {
+        if (hasOrdinal(s)) {
+            String[] parts = s.split("/");
+            return new AppendField(parts[0], ORD + Integer.valueOf(parts[1]));
```

Comment:
That's odd. I thought that the ordinal is something like a id for the type of field and an `AppendField` instance always has the same ordinal. Maybe come up with another name? What's the intention behind it?